app/assets/javascripts/calendars/holidays.es6

```diff
@@ -0,0 +1,40 @@
+/* global Calendar */
+{
+  'use strict';
+
+  class HolidaysCalendar extends Calendar {
+    constructor(el, config = {}) {
+      const calendarConfig = $.extend(true, {
+        defaultView: 'agendaWeek',
+        weekends: false,
+        columnFormat: 'ddd D/M',
+        slotDuration: '00:30:00',
+        eventBorderColor: '#000',
+        header: {
+          'right': 'agendaWeek agendaThreeDay agendaDay today prev,next'
+        },
+        views: {
+          agendaThreeDay: {
+            type: 'agenda',
+            duration: { days: 3 }
+          }
+        }
+      }, config);
+
+      super(el, calendarConfig);
+
+      this.init();
+    }
+
+    handleEventData(data) {
+      this.$el.fullCalendar('addEventSource', data);
+    }
+
+    init() {
+      $.getJSON('/holidays', this.handleEventData.bind(this));
```

Comment:
So you should be able to use the `event` attribute of the full calendar config as a string to the 'API' end point where it can grab events